baseband/vlbi_base/payload.py

```diff
@@ -37,12 +37,14 @@ class VLBIPayloadBase(object):
     """
     # Possible fixed payload size.
     _size = None
-    # Default type for encoded data
+    # Default type for encoded data.
     _dtype_word = np.dtype('<u4')
     """Default for words: 32-bit unsigned integers, with lsb first."""
     # To be defined by subclasses.
     _encoders = {}
     _decoders = {}
+    # Placeholder for sample shape named tuple.
+    _sample_shape_cls = tuple
 
     def __init__(self, words, bps=2, sample_shape=(), complex_data=False):
         self.words = words
```

Comment:
I think we might as well pass `sample_shape` through `self._sample_shape_cls` here.

Reply:
So, I'd still argue to here do
```
self.sample_shape = self._sample_shape_cls(*sample_shape)
```
(with the above definition, this will always work)